lib/components/dashboard/index.tsx

```diff
@@ -92,56 +90,6 @@ const Dashboard: React.FC<Props> = ({ mode }) => {
   // State Setup
   const [showNotif, setShowNotif] = useState<boolean>(true);
   const [formPopup, setFormPopup] = useState<boolean>(false);
-  // const [showMobileMenu, setShowMobMenu] = useState(false);
```

Comment:
hi fyona~ i don't think this commented out code (lines 95 - 144) is related to roadmap, do you mind re-adding it for now?

Reply:
<img width="513" alt="Screen Shot 2023-04-16 at 4 30 47 PM" src="https://user-images.githubusercontent.com/77924721/232340399-aa135079-d287-4581-8297-b64587fbb5f3.png">
Hi Josephine! When I add it back it says the values are unused so I'm not sure whether I need to add it back